docs/en/deployment/hadoop_java_sdk.md

```diff
@@ -421,6 +421,7 @@ CREATE TABLE IF NOT EXISTS person
 
 2. Use the following sample code to verify:
 
+<!-- autocorrect: false -->
```

Comment:
is it possible to configure autocorrect to ignore all code snippets? rather than manually adding comment notations for individual code snippets

Reply:
This is a bug of autocorrect, I've report it to the upstream. For now, it's a compromise approach.